yefer-garcia/tarea-10-globant/src/App.js

```diff
@@ -0,0 +1,107 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Footer from "./components/Footer";
+import Navbar from "./components/Navbar";
+import Home from "./pages/Home";
+import "./css/componentes.css";
+import "./css/quiz.css";
+import Quiz1 from "./pages/Quiz1";
+import Quiz2 from "./pages/Quiz2";
+import Quiz3 from "./pages/Quiz3";
+import Quiz4 from "./pages/Quiz4";
+import Quiz5 from "./pages/Quiz5";
+import Quiz6 from "./pages/Quiz6";
+import Youwin from "./pages/YouWin";
+import NotFound from "./components/NotFound";
+import Lost from "./pages/Lost";
+import { useState } from "react";
+
+function App() {
+  const [points, setPoints] = useState(0);
+  const [vidas, setVidas] = useState(3);
+
+  return (
+    <BrowserRouter>
+      <Navbar setpuntos={setPoints} setvidas={setVidas} />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route
+          path="/firts-Quiz"
+          element={
+            <Quiz1
+              puntos={points}
+              vidas={vidas}
+              setpuntos={setPoints}
+              setvidas={setVidas}
+            />
+          }
+        />
+        <Route
+          path="/second-Quiz"
+          element={
+            <Quiz2
```

Comment:
@Yefer-Garcia , please implement only one compoenent **_Quiz_**, that should take additional to the current parameters:

```suggestion
            <Quiz  
              options={[]}
               answer={} 
```

Or something like that , your are repeating a lot of code with the current architecture, which is not maintanable.